packages/lesswrong/server/languageModels/llmApiWrapper.ts

```diff
@@ -176,7 +176,19 @@ export async function sendMessagesToLlm<T extends SendLLMMessagesArgs>(args: T):
     const responseContent = firstContentBlock.input;
     const validatedTerm = zodParser.safeParse(responseContent);
     if (!validatedTerm.success) {
-      throw new Error('Invalid tool use response from Anthropic!');
+      // eslint-disable-next-line no-console
+      console.log('Failed to parse tool use response from Anthropic with primary schema, trying backup...', { responseContent });
+      // lol
```

Comment:
i'm so sorry

Reply:
what's actually supposed to happen here?